src/L.Control.Locate.js

```diff
@@ -468,6 +468,7 @@ You can find the project at: https://github.com/domoritz/leaflet-locatecontrol
         _activate: function() {
             if (!this._active) {
                 this._map.locate(this.options.locateOptions);
+                this._map.fire('locatestart', this);
```

Comment:
Since this function is called `activate` do you think it would be better to call the even `locateactivate`? Similarly for the other one.

Reply:
Hmm, I was originally basing this on the `start` and `stop` functions for analogy (which in turn call `activate` and `deactivate`), but I think you're right and it would be more accurate and representative to rename the events.